src/include/sof/audio/pipeline.h

```diff
@@ -30,6 +30,9 @@ struct ipc_msg;
 /* Pipeline status to stop execution of current path */
 #define PPL_STATUS_PATH_STOP	1
 
+/* Pipeline scheduled in another thread other than ipc thread */
+#define PPL_STATUS_SCHEDULED	1
```

Comment:
this collides with PPL_STATUS_PATH_STOP value

Reply:
sure. let's use 2